AirCasting/Settings/MyAccountViewSignOut.swift

```diff
@@ -12,6 +12,7 @@ struct MyAccountViewSignOut: View {
     @Injected private var networkChecker: NetworkChecker
     @Injected private var logoutController: LogoutController
     @Injected private var deleteController: DeleteAccountController
+    @Injected private var measurementStreamStorage: MeasurementStreamStorage
```

Comment:
please move storage logic to `LogoutController` (it will perhaps require some thinking because of the alert requirement, but please try. Ping me if problematic)